modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -70,7 +70,7 @@ page import="com.liferay.server.admin.web.internal.display.context.ServerDisplay
 page import="com.liferay.server.admin.web.internal.display.context.ViewPortalPropertiesDisplayContext" %><%@
 page import="com.liferay.server.admin.web.internal.display.context.ViewSystemPropertiesDisplayContext" %><%@
 page import="com.liferay.server.admin.web.internal.image.ImageMagickUtil" %><%@
-page import="com.liferay.server.admin.web.internal.scripting.util.ServerScriptingUtil" %><%@
+page import="com.liferay.server.admin.web.internal.scripting.ServerScripting" %><%@
```

Comment:
Should it be `ServerScriptingUtil` instead of `ServerScripting`? 🤔

Reply:
yeah, in both JSP files, sorry